src/Libraries/UnitsUI/NodeViewCustomizations.cs

```diff
@@ -17,7 +17,7 @@
 
 namespace UnitsUI
 {
-    [Obsolete("This abstract class will be removed in Dynamo 3.0 - please use the StringInputNodeViewCustomization")]
+    [Obsolete("This abstract class will be removed in a future version of Dynamo - please use the StringInputNodeViewCustomization")]
     public abstract class MeasurementInputBaseNodeViewCustomization : INodeViewCustomization<MeasurementInputBase>
```

Comment:
I think I added a comment previously about considering getting rid of all this old unit stuff... I guess that could break graphs even though these have obsolete for a bit. 
Thoughts @saintentropy @Amoursol ?

Reply:
@mjkkirschner we're trying to limit damage to package authors and this is OOTB functionality. We should err on the side of caution here - I like your proposed messaging changes.

Reply:
Out of scope for now.